### ISSUE TITLE
TiUP import failed to read a directory 

### ISSUE THREAD

**Issue by oscar-zyh** [pingcap/tiup#399]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?
import an tidb-ansible to tiup 
`tiup cluster import -d /data/deploy/tidb-staging`

2. What did you expect to see?
import successfully

3. What did you see instead?
```
...
+ [ Serial ] - CopyFile: remote=xxx:/data/deploy/conf/tikv.toml, local=/home/tidb/.tiup/storage/cluster/clusters/staging-cluster/ansible-imported-configs/tikv-xxx-20160.toml
+ [ Serial ] - CopyFile: remote=xxx:/data/deploy/conf/tidb.toml, local=/home/tidb/.tiup/storage/cluster/clusters/staging-cluster/ansible-imported-configs/tidb-xxx-4000.toml
Finished copying configs.

Error: read /data/deploy/tidb-staging: is a directory
```

4. What version of TiUP are you using (`tiup --version`)?
v1.0.1

**Comment by lonng** [comment]
Are you sure that the directory `/data/deploy/tidb-staging` is your ansible directory?

**Comment by lonng** [comment]
Closed via #410
